File: wiz-shop-java/src/main/java/com/example/wizshopjava/api/config/WebConfig.java
package com.example.wizshopjava.api.config;

import java.util.List;

import org.springframework.context.annotation.Configuration;
import org.springframework.web.method.support.HandlerMethodArgumentResolver;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;

import com.example.wizshopjava.api.jwtauth.RequestMemberResolver;

import lombok.RequiredArgsConstructor;

@Configuration
@RequiredArgsConstructor
public class WebConfig implements WebMvcConfigurer {

	private final RequestMemberResolver requestMemberResolver;

	@Override
	public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers) {
		resolvers.add(requestMemberResolver);
	}
}

File: wiz-shop/src/main/kotlin/com/example/wizshop/api/config/WebConfig.kt
package com.example.wizshop.api.config

import com.example.wizshop.api.jwtauth.RequestMemberResolver
import org.springframework.context.annotation.Configuration
import org.springframework.web.method.support.HandlerMethodArgumentResolver
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer

@Configuration
class WebConfig(
    private val requestMemberResolver: RequestMemberResolver
) : WebMvcConfigurer {

    override fun addArgumentResolvers(resolvers: MutableList<HandlerMethodArgumentResolver>) {
        resolvers.add(requestMemberResolver)
    }
}